src/index.html

```diff
@@ -3,12 +3,61 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav__logo">
+        <img class="logo" src="/src/images/logo.png"
+         alt="logo"
+      ><a href="#index.html"></a></img>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple.html" class="nav__link is-active">apple</a>
```

Comment:
no need to add `html`, fix it everywhere
```suggestion
            <a href="#apple" class="nav__link is-active">apple</a>
```